common-canvas/form/form-v4-schema.json

```diff
@@ -590,6 +590,14 @@
         "className": {
           "description": "Optional class name to set for this control",
           "type": "string"
+        },
```

Comment:
Don't we need a new control type added to form schema?

Reply:
added